packages/vue-language-integration/src/index.ts

```diff
@@ -4,7 +4,7 @@ export const languageId = 'vue';
 export const extension = '.vue';
 
 export function toTSX(code: string, className: string): string {
-	let result = `export default function ${className}__AstroComponent_(): any {}`;
+	let result = `export default function ${className}__AstroComponent_(_props: Record<string, any>): any {}`;
```

Comment:
Unrelated to the changes, this fix the language server complaining about props even in cases where it couldn't tell what props were used. Now, when we can't parse the props, we'll just assume that everything is okay